docs/persistent-volumes.md

```diff
@@ -0,0 +1,170 @@
+A Persistent Volume Claim (PVC) is a request for storage by a user.  It is similar to a pod.  Pods consume node resources and PVCs consume PV resources.  Pods can request specific levels of resources (CPU and Memory).  Claims can request specific size and access modes (e.g, can be mounted once read/write or many times read-only).  
+
+Please see the [detailed walkthrough with working examples](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/persistent-volumes).
+
+
+## Lifecycle of a volume and claim
+
+PVs are resources in the cluster.  PVC are requests for those resources and also act as claim checks to the resource.  The interaction between PVs and PVCs follows this lifecycle:
+
+### Provisioning
+	
+The volume is created by an administrator.  It becomes a cluster resource available for consumption.
+
+### Binding
+
+A persistent volume claim is created by a user requesting a specific amount of storage and with certain access modes.  There is a process watching for new claims that binds them to an available volume if a match is available.  The user will always get at least what they asked for, but the volume may be in excess of what was requested.
+
+Claims will remain unbound indefinitely if a matching volume does not exist.  Claims will be bound as matching volumes become available.  For example, a cluster provisioned with many 50Gi volumes would not match a PVC requesting 100Gi.  The PVC can be bound when a 100Gi PV is added to the cluster.
+
+### Using
+
+Pods use their claim as a volume.  The cluster uses the claim to find the bound volume bound and mounts that volume for the user.  For those volumes that support multiple access modes, the user specifies which mode desired when using their claim as a volume in a pod.
+
+### Releasing
+
+When a user is done with their volume, they can delete their claim which allows reclamation of the resource.  The volume is considered "released" when the claim is deleted, but it is not yet available for another claim.  The previous claimant's data remains on the volume which must be handled according to policy.
+
+### Reclaiming
+
+A persistent volume's reclaim policy tells the cluster what to do with the volume after it's released.  Currently, volumes can either be Retained or Recycled.  Retention allows for manual reclamation of the resource.  For those volume plugins that support it, recycling performs a basic scrub ("rm -rf /thevolume/*") on the volume and makes it available again for a new claim.
+
+## Types of Persistent Volumes
+
+Persistent volumes are implemented as plugins.  Kubernetes currently supports the following plugins:
+
+* GCEPersistentDisk
+* AWSElasticBlockStore
+* NFS
+* ISCSI
+* RBD
+* Glusterfs
+* HostPath (single node testing only)
+
+
+## Persistent Volumes
+
+Each PV contains a spec and status, which is the specification and status of the volume.  
+
+```
+
+  apiVersion: v1
+  kind: PersistentVolume
+  metadata:
+    name: pv0003
+  spec:
+    capacity:
```

Comment:
Aside - are we ever going to have more than one resource here?  It would be cleaner to just say "Capacity: 5Gi"